Dockerfile

```diff
@@ -28,4 +28,4 @@ COPY --from=build-env /go/src/github.com/ChainSafe/ethermint/build/emintcli /usr
 EXPOSE 26656 26657 1317
 
 # Run emintd by default, omit entrypoint to ease using container with emintcli
-CMD ["emintd"]
+ENTRYPOINT ["/bin/bash", "-c"]
```

Comment:
I don't understand the diff here with `CMD ["emintd"]` can you add a code comment?

Reply:
CMD is used when there are no args passed the the docker run command (i.e. `docker run <image>). We pass args like `testnets [flags] ...` so we would need entrypoint. 

this might be a helpful explanation
https://stackoverflow.com/a/44201230